README.md

```diff
@@ -175,13 +174,13 @@ The widget registry allows you to lazy instantiate widgets.
 
 ### Writing Custom Widgets
 
-The `createWidgetBase` class provides the functionality needed to create Custom Widgets.
+The `WidgetBase` class provides the functionality needed to create Custom Widgets.
 This functionality includes caching and widget lifecycle management.
 
-The `createWidgetBase` class is available from the `@dojo/widget-core/createWidgetBase` package.
+The `WidgetBase` class is available from the `@dojo/widget-core/WidgetBase ` package.
 
 ```ts
-import { createWidgetBase } from '@dojo/widget-core/createWidgetBase';
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
```

Comment:
It is named and default. So both are correct, I guess we need to decide which we use consistently through out the documentation.